sysutils/istioctl/Portfile

```
@@ -6,9 +6,8 @@ PortGroup           golang 1.0
 name                istioctl
 revision            0
 
-go.setup            github.com/istio/istio 1.17.1
-# Delete this on next update to use golang PortGroup's default ('archive')
-github.tarball_from tarball
+go.setup            github.com/istio/istio 1.28.0
+github.tarball_from archive
```

Let's add `revision`:

```suggestion
github.tarball_from archive
revision            0
```

---

done, thanks.

ps: it would be nice to get the warnings from the lint nitpick..